assets/stylesheets/shared/_color-schemes.scss

```diff
@@ -320,45 +314,12 @@
 		--color-accent-800: #{$muriel-orange-800};
 		--color-accent-800-rgb: #{hex-to-rgb( $muriel-orange-800 )};
 		--color-accent-900: #{$muriel-orange-900};
-		--color-accent-900-rgb: #{hex-to-rgb( $muriel-orange-900 )};
```

Comment:
Why is this line deleted? Shouldn't we keep it for the Classic Blue theme?

Reply:
It's just GitHub being weird, it's here too so isn't actually being deleted. :) 

https://github.com/Automattic/wp-calypso/blob/cfa945f83480b75b152cf4699bcd7584719d092a/assets/stylesheets/shared/_color-schemes.scss#L317

Reply:
Oh I see that it's added again down below...

Reply:
Yep!